spec/services/action_builder_spec.rb

```diff
@@ -186,98 +186,4 @@ def initialize(params)
     end
   end
 
-  describe 'permitted_keys' do
-    let(:mab) { MockActionBuilder.new(page_id: page.id, email: member.email) }
-
-    it 'returns symbols' do
-      expect(mab.permitted_keys.map(&:class).uniq).to eq [Symbol]
-    end
-
-    it 'does not include the id' do
-      expect(mab.permitted_keys).not_to include(:id)
-    end
-
-    it 'includes all the other keys of member' do
-      expect(mab.permitted_keys)
-        .to include(
-          :email,
-          :country,
-          :first_name,
-          :last_name,
-          :city,
-          :postal,
-          :title,
-          :address1,
-          :address2,
-          :actionkit_user_id
-        )
-    end
-  end
-
-  describe 'filtered_params' do
-    let(:params) do
-      {
-        email: 'silly@billy.com',
-        country: 'US',
-        first_name: 'Silly',
-        last_name: 'Billy',
-        city: 'Northampton',
-        postal: '01060',
-        address1: '10 Coates St.',
-        address2: ''
-      }
-    end
-
-    describe 'passes all' do
-      it 'keys as symbols' do
-        mab = MockActionBuilder.new(params)
-        expect(mab.filtered_params).to eq params
-      end
-
-      it 'keys as strings' do
-        mab = MockActionBuilder.new(params.stringify_keys)
-        expect(mab.filtered_params).to eq params
-      end
-
-      it 'keys with indifferent access' do
-        mab = MockActionBuilder.new(params.with_indifferent_access)
-        expect(mab.filtered_params).to eq params
-      end
-
-      it 'keys as action parameters' do
-        mab = MockActionBuilder.new(ActionController::Parameters.new(params))
-        expect(mab.filtered_params).to eq params
-      end
-    end
-
-    describe 'filters irrelevant' do
-      let(:porky_params) { params.merge(page_id: page.id, form_id: '3', blerg: false, akid: '1234.514.lQVxcW') }
-
-      it 'keys as symbols' do
-        mab = MockActionBuilder.new(porky_params)
-        expect(mab.filtered_params).to eq params
-      end
-
-      it 'keys as strings' do
-        mab = MockActionBuilder.new(porky_params.stringify_keys)
-        expect(mab.filtered_params).to eq params
-      end
-
-      it 'keys with indifferent access' do
-        mab = MockActionBuilder.new(porky_params.with_indifferent_access)
-        expect(mab.filtered_params).to eq params
-      end
-
-      it 'keys as action parameters' do
-        mab = MockActionBuilder.new(ActionController::Parameters.new(porky_params))
-        expect(mab.filtered_params).to eq params
-      end
-
-      it 'but passes them through to form_data' do
-        mab = MockActionBuilder.new(porky_params)
-        expect { mab.build_action }.to change { Action.count }.by 1
-        expect(Action.last.form_data).to match a_hash_including(params.stringify_keys)
```

Comment:
Good catch! I didn't mean to remove this. I've added it back in a slightly different format.